into/backends/spark.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import division, print_function, absolute_import
+
+
+class Dummy(object):
+    pass
+
+try:
+    from pyspark import SparkContext
+    import pyspark
+    from pyspark import RDD
+    from pyspark.rdd import PipelinedRDD
+    from pyspark.sql import SchemaRDD, SQLContext
+    RDD.min
+except (AttributeError, ImportError):
+    SchemaRDD = PipelinedRDD = RDD = SparkContext = SQLContext = Dummy
+    pyspark = Dummy()
+
+
+from datashape import var
+
+from .. import convert, append, discover
+
+
+@append.register(SparkContext, list)
+def list_to_spark_context(sc, seq, **kwargs):
+    return sc.parallelize(seq)
+
+
+@append.register((SparkContext, SQLContext), object)
```

Comment:
That is what will happen, I've made it so that there's only a single `SQLContext` `append` function